app/src/main/java/com/pajato/android/gamechat/exp/ChessPiece.java

```diff
@@ -1,39 +1,68 @@
 package com.pajato.android.gamechat.exp;
 
-import android.util.SparseArray;
-
+import com.google.firebase.database.Exclude;
+import com.google.firebase.database.IgnoreExtraProperties;
 import com.pajato.android.gamechat.R;
 
-import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+import static android.R.attr.name;
 
 /**
  * A simple P.O.J.O. class that keeps track of a chess pieces type and the team it is on.
  */
-class ChessPiece {
-    static final int KING = 0;
-    static final int QUEEN = 1;
-    static final int BISHOP = 2;
-    static final int KNIGHT = 3;
-    static final int ROOK = 4;
-    static final int PAWN = 5;
+@IgnoreExtraProperties
+public class ChessPiece {
+
+    public enum ChessTeam {
+        NONE, PRIMARY, SECONDARY
+    }
 
-    static final int PRIMARY_TEAM = 10;
-    static final int SECONDARY_TEAM = 20;
+    public enum PieceType {
```

Comment:
I would be inclined to make this a one liner: `public enum PieceType {NONE, KING, ... PAWN` as it does not sacrifice readability or clarity.